app/src/main/java/sensors_in_paradise/sonar/page2/LoggingManager.kt

```diff
@@ -181,6 +194,7 @@ class LoggingManager(
         resolveMissingFields {
             moveTempFiles(personTV.text.toString(), recordingEndTime)
             labelTV.text = ""
+            personTV.text = ""
```

Comment:
This line was removed in order to not reset the person after every recording.

Reply:
Well spotted! Fixed now